vamb/__main__.py

```diff
@@ -1160,6 +1161,11 @@ def try_from_common(cls, common: BinnerCommonOptions):
             )
         else:
             return None
+            
+def write_clusters_to_file(file_handle, file_path, clusters, to_file):
```

Comment:
Add argument types, similar to how the other functions are (`clusters: dict[str, set[str]],` etc)